### PR TITLE
Add docs for Visual Studio

### PR DESCRIPTION
**Pull request by MaedahBatool**
Docs for Visual Studio extension for Cody.

### CODE REVIEW

**Comment by MaedahBatool** [comment]
@kalanchan can I please get your review.

File: docs/cody/clients/feature-reference.mdx [hunk 1]
```diff
@@ -4,28 +4,27 @@
...
+| **Context Selection**                    |             |               |                      |                   |         |
+| Single-repo context                      | ✅           | ✅             | ✅                    | ✅                 | ❌       |
+| Multi-repo context                       | ❌           | ❌             | ✅ (public code only) | ❌                 | ❌       |
+| Local context                            | ✅           | ✅             | ❌                    | ✅                 | ✅       |
+| OpenCtx context providers (experimental) | ✅           | ❌             | ❌                    | ❌                 | ❌       |
+| **Prompts and Commands**                 |             |               |                      |                   |         |
+| Access to prompts and Prompt library     | ✅           | ✅             | ✅                    | ✅                 | ❌       |
+| Edit code                                | ✅           | ✅             | ❌                    | ✅                 | ❌       |
+| Generate unit test                       | ✅           | ✅             | ❌                    | ✅                 | ❌       |
```
Comment (kalanchan) [inline docs/cody/clients/feature-reference.mdx:26]:
```suggestion
| Generate unit test                       | ✅           | ✅             | ❌                    | ❌                 | ❌       |
```

**Review by kalanchan — CHANGES_REQUESTED** [review]

**Comment by kalanchan** [comment]
thanks maedah! looks good, just requested a couple changes

File: docs/cody/clients/feature-reference.mdx (hunk 1 above)
Comment (MaedahBatool) [reply docs/cody/clients/feature-reference.mdx:26]: Yeah this will go away with Nov 25th release.

**Pull request merged**